PrimerasPracticasJS/SacarIVA.js

```diff
@@ -0,0 +1,21 @@
+/*FUNCION PARA SACAR EL IVA*/
+
+let precio = 0
+let iva = 0
+function AplicarIVA(precio,iva){
+
+    if (iva != null) {
+        let sacarIVA = (iva * precio) / 100
+        let PrecioConIVA = sacarIVA + precio
+
+        return(PrecioConIVA)
```

Comment:
no son necesarios los parentisis 
```suggestion
        return PrecioConIVA
```